helm/hpcc/templates/eclccserver.yaml

```diff
@@ -3,6 +3,7 @@ EclccServer configmap
 Pass in dict with root and me
 */}}
 {{- define "hpcc.eclccServerConfigMap" -}}
+{{- $compileJobName := printf "compile-%%jobname" }}
```

Comment:
This change doesn't look necessary, the printf isn't acutally doing anything here (no arguments).
Could revert this and line 27 to way it was.